images/fedora/f32/README.md

```diff
@@ -54,7 +54,7 @@ This will create a container called `fedora-toolbox-<version-id>`.
 ### Enter the toolbox:
 ```console
 [user@hostname ~]$ toolbox enter
-⬢[user@toolbox ~]$
+⬢[user@fedora-toolbox-<version-id> ~]$
```

Comment:
This shows the problem with using the name of the container in the hostname. :)

As I mentioned [elsewhere](https://github.com/containers/toolbox/issues/98#issuecomment-1318592340), it makes shell prompts with `\h`, which is often the case, too long.  Users who are fine with longer prompts can customize it on their own using [metadata](https://github.com/containers/podman/issues/6192) from `/run/.containerenv`, but we shouldn't do that through the hostname because of how common it is for default prompts to have `\h` in them.

*For what it's worth, https://github.com/containers/toolbox/issues/98 is the issue about making it easier to distinguish one Toolbx container from another.*

Reply:
I am okay with  leaving this as `toolbox` for this change - will pick this up again and make this change

however my shell prompt with `\u @ \h` crops the FQDN at the first component so it doesn't get long, example:

```
[akdev@vancouver ~]$ export PS1='\u @ \h > '
akdev @ vancouver > hostname
vancouver.hq.akdev.xyz
```

with FQDN = `vancouver.hq.akdev.xyz` the prompt shows `vancouver`